packages/repository/test/unit/decorator/repository-with-di.ts

```diff
@@ -16,57 +16,78 @@ import {
   DataSourceType,
 } from '../../../';
 
-class MyController {
-  constructor(@repository('noteRepo') public noteRepo: Repository<Entity>) {}
-}
-
 describe('repository class', () => {
   let ctx: Context;
 
   before(function() {
```

Comment:
Can you refactor function to a named function and call that from the before. Something like `givenCtx`. 

See our test guidelines: http://loopback.io/doc/en/contrib/style-guide.html#hooks

Reply:
good find